vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,159 +2,94 @@
 ``` sh
-  docker-compose -f src/main/docker/keycloak.yml up
+ent bundle run conference-table
 ```
+If you are not logged in, you'll be redirected to do so. Log in using the following credentials. 
+* Username: user 
+* Password: user\
+Once logged in, you will see the table widget with some generated data.
 
->Note: If you need to install 
-Docker Compose, follow this guide:
-<https://docs.docker.com/compose/install/>
-
-
-### Start the Microservice
-
-1.  Start the generated microservice with the command:
-
-        ./mvnw
-
-#### Notes
-1. If you want to reset the widget data (for example, if you deleted all rows from the table widget), and you selected "H2 with disk-based persistence" during microservice generation, you can delete the target folder, restart the microservice, and the data will be regenerated.
-2. The `service-url` variable is the microservice API URL.
-
-### Start the Table Widget
-
-1.  Go to the table widget folder in your project:
-
-        cd ui/widgets/YOUR-ENTITY-NAME/tableWidget
-
-2.  Install and start your widget with the command:
-
-        npm install && npm start
-
-3.  When the widget starts, a browser window opens the widget URL
-
-4.  If you’re not logged in, you'll be redirected to the login page.
-
-5.  Log in using:
-
-        Username: user
-        Password: user
-
-6.  Once logged in, you can see the table widget with generated data.
-
-### Start the Form Widget
-
-1.  If you are already running a widget, click `Ctrl+C` to end the process
-
-2.  Go to the form widget folder in your project:
-
-        cd ui/widgets/YOUR-ENTITY-NAME/formWidget
-
-3.  Install and start your widget with the command:
-
-        npm install && npm start
-
-4.  When the widget starts, a browser window opens the widget URL 
-
-5.  If you’re not logged in, you'll be redirected to the login page
-
-6.  Log in using:
-
-        Username: user
-        Password: user
 
-7.  Once logged in, you will see the widget form.
+### Run the conference-form MFE
 
-#### Form Widget notes:
+1.  If you are still running the previous micro frontend, click `Ctrl+C` to end the process
 
-If you want to modify a different row in the database, edit the `index.html` file in the public folder `ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public`. Change the `id` attribute in this line:
-```
-   <my-entity-form service-url="%REACT_APP_SERVICE_URL%" id="1" />
+2. Start up the conference-form MFE 
+``` sh
+ent bundle run conference-form
 ```
-### Start the Details Widget
-
-1. If you are already running a widget, click `Ctrl+C` to end the process
-
-2. Go to the details widget folder in your project:
-
-        cd ui/widgets/YOUR-ENTITY-NAME/detailsWidget
-
-3. Install and start your widget with the command:
+The form to enter the name and location for conferences shoud open in your browser. You may enter and save new data to see it in the other MFEs.
```

Comment:
it looks like this is really muddy territory and singular usage is gaining traction. i actually prefer singular in most cases because plural sounds kind of pretentious